tests/utils/profilehandler/data_preparation.go

```diff
@@ -254,14 +254,25 @@ func PrepareAccountRoles(client *rosacli.Client,
 	if err != nil {
 		return
 	}
-	accRoleList, output, err := client.OCMResource.ListAccountRole()
-	if err != nil {
-		err = fmt.Errorf("error happens when list account-roles, %s", output.String())
-		return
+	var accoutRoles *rosacli.AccountRolesUnit
+	var interErr error
+	maxRetries := 3
+	for i := 0; i < maxRetries; i++ {
+		accRoleList, output, err := client.OCMResource.ListAccountRole()
+		if err != nil {
```

Comment:
```suggestion
		if err != nil &&strings.Contains(err.Error(), "cannot be found"){
```
This can reduce the `if`/`else` in code

Reply:
@xueli181114 There are three condition here,
- err  != nil and strings.Contains(err.Error(), "cannot be found")
- err != nil and !strings.Contains(err.Error(), "cannot be found")
- err = nil

The way as you suggest also needs two if-else as the current one, from my side, it is more clear with current implement. Anyway, I updated as you suggest, please take a look again.